packages/editor/src/components/widgets/table/row/MessageRow.tsx

```diff
@@ -4,15 +4,17 @@ import { forwardRef } from 'react';
 import type { MessageTextProps } from '../../message/Message';
 import { MessageText } from '../../message/Message';
 
-export type MessageRowProps = MessageTextProps & ComponentProps<'tr'> & { colSpan: number };
+export type MessageRowProps = MessageTextProps & ComponentProps<'tr'> & { colSpan: number; selectable?: boolean };
 
 export const MessageRow = forwardRef<HTMLTableRowElement, MessageRowProps>(
-  ({ message, children, className, colSpan, ...props }, forwardRef) => {
+  ({ message, children, className, colSpan, selectable, ...props }, forwardRef) => {
     return (
       <>
-        <tr ref={forwardRef} className={`row ${message ? `row-${message.severity.toLocaleLowerCase()}` : ''} ${className}`} {...props}>
-          {children}
-        </tr>
+        {!selectable && (
+          <tr ref={forwardRef} className={`row ${message ? `row-${message.severity.toLocaleLowerCase()}` : ''} ${className}`} {...props}>
+            {children}
+          </tr>
+        )}
```

Comment:
not exaclty the way I would do it...
I would split this up into two components one for each row, which then can be reused everywhere else...
And still the select row misses the row-<severiy> class right? so it will not show a red border around the row if something is wrong

Reply:
I'm not entirely sure if I've understood your idea correctly. Do you want to modify the MessageRow component so that it only consists of the message row, and create a new second component that reflects the select row, but additionally includes the CSS class of MessageRow? And than reuse both components within the SelectableValidationRow?

For now, I've adjusted the SelectRow to set the corresponding CSS class for the red border. Otherwise, we can briefly discuss how you envisioned it with the two components.

Reply:
let's have a look at it together tomorrow